assets/src/edit-story/components/canvas/carousel/index.js

```diff
@@ -102,11 +139,12 @@ function Carousel() {
 
   useResizeEffect(
     listRef,
-    () => {
+    (carousel) => {
```

Comment:
So, this handler is invoked with `{width, height}`.  Maybe `currentSize` would be a better name for this variable?

I thought from the name it was a reference to the node, but that's already covered by `listRef.current`.

Reply:
First I went with `carouselSize` but because of `no-shadow` I had to come up with something else, maybe let's go with mix of both: `currentCarouselSize` 🤓